### PR TITLE
Add LDAP login plugin by datamints

### PR DESCRIPTION
**Pull request by sebastiangreger**
Found in Forum thread https://forum.getkirby.com/t/ldap-login-into-panel/14945

### CODE REVIEW

File: content/plugins/datamints/kirby-plugin_ldap/plugin.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Title: Datamints Kirby-Plugin Ldap
```
Comment (lukasbestle) [inline content/plugins/datamints/kirby-plugin_ldap/plugin.txt]: Your developer name will be automatically printed below the plugin name. And that it's a Kirby plugin is obvious as the plugin will be listed in our plugin directory.

Suggestion:

```suggestion
Title: LDAP login
```

**Review by lukasbestle — COMMENTED** [review]
Thank you for your contribution, both for the plugin and for adding it to our plugin directory.
One minor comment before it can be merged:

**Comment by sebastiangreger** [comment]
Hi. This is not my own plugin, but authored by @datamints -- I only added it to the plugin repo based on [their request](https://forum.getkirby.com/t/ldap-login-into-panel/14945/20) in the linked forum thread ...and because I consider it worth to be found.

Hereby pinging the original author, as I cannot really make that decision on your suggestion above.

**Comment by lukasbestle** [comment]
@sebastiangreger Ah, alright. Thanks!

**Comment by lukasbestle** [comment]
Because there was no response by the developer so far, I have changed the title as suggested and merged the PR. We can still change the title later if needed.

@sebastiangreger Thanks for your PR! 👍

**Comment by datamints** [comment]
Hi there,
thank you very much for your work.
Sorry that I haven't answered anything of this yet.
Hopefully I will find the time to maintain the plugin soon.
To make the plugin findable was indeed my intention, I just didn't know how.
If there is anything more for me to do, just keep pinging this github account, it will be forwarded to me.

**Pull request merged**